specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/streams/application/merged/merged.fetch.filter.none/client.rpt

```diff
@@ -102,6 +102,8 @@ read zilla:data.ext ${kafka:matchDataEx()
                              .build()}
 read "Hello, world #B3"
 
+read notify PARTITION_ZERO_OFFSET_FOUR
```

Comment:
Suggest we change the names of these barriers to help readability, such as `RECEIVED_MESSAGE_B3`, etc.